packages/core/src/builder.class.ts

```diff
@@ -1,3 +1,4 @@
+import 'custom-event-polyfill';
```

Comment:
I actually vaguely recall us using this then removing it due to a problem. can you check the source code to see if this is identical to what we used in the webcomponents? or possibly check the git history as to when we changed this over to be manually copy + pasted and edited

my vague recollection was it defining the CustomEvent even if it already was defined or something. or not being compatible server side (not checking for window existing) first

Reply:
ok in this case will use the old polyfill